lms/templates/learner_dashboard/program_details_sidebar.underscore

```diff
@@ -8,14 +8,61 @@
     <% } %>
 </aside>
 <aside class="aside js-course-certificates"></aside>
+<% if (isSubscriptionEligible) { %>
+<aside class="aside js-subscription-info subscription-info">
+    <h2 class="divider-heading">
+        <%- gettext(
+            hasActiveTrial
+            ? 'Trial subscription'
+            : subscriptionState === 'active'
+            ? 'Active subscription'
+            : 'Inactive subscription'
+        ) %>
+    </h2>
+    <div class="sidebar-section">
+        <div class="subscription-section">
+            <p class="my-0">
+                <%= HtmlUtils.interpolateHtml(
+                    gettext((
+                        subscriptionState === 'active'
+                        ? 'View your receipts or modify your subscription'
+                        : subscriptionState === 'inactive'
+                        ? 'Restart your subscription for {subscriptionPrice}/month. Your payment history is still available'
+                        : 'If you had a subscription previously, your payment history is still available'
+                    ).concat(' on the {a_start}Orders and subscriptions{a_end} page')),
+                    {
+                        subscriptionPrice,
+                        a_start: HtmlUtils.HTML(`<a class="subscription-link" href="${manage_subscription_url}">`),
+                        a_end: HtmlUtils.HTML('</a>'),
+                    }
+                ) %>
+            </p>
+            <p class="my-0">
+                <%= HtmlUtils.interpolateHtml(
+                    gettext(
+                        'Need help? Check out the {a_start}Learner help centre{span_start}{icon}{span_end}{a_end} to troubleshoot issues or contact support '
```

Comment:
nit: If this is linking to the official support page (https://support.edx.org/hc/en-us) then I believe this should read `Learner Help Center` with "Help" and "Center" capitalized. I feel like this could be simplfied to something like `Need help? Check out the Learner Help Center to contact support`.

Reply:
about the capitalization part the design team confirmed all the text is capitalised as per edx standards, same for the text
I will forward your suggestion to them though :)

Reply:
Hm. It still doesn't seem right to me, but thanks for engaging the design team.